README.md

```diff
@@ -67,7 +67,7 @@ While the Agent and Collector share the same binary, the configuration between
 the two may differ depending on requirements (e.g. queue size and feature-set
 enabled).
 
-![deployment-models](images/opentelemetry-service-deployment-models.png)
+![deployment-models](https://imgur.com/Tj384ap)
```

Comment:
Doesn't seem to be working for me.
This is what I see:
![image](https://user-images.githubusercontent.com/4194920/60613092-2bfb7380-9d98-11e9-8065-aae1503314d6.png)

Reply:
Done.